### PR TITLE
new licensing

### PR DESCRIPTION
**Pull request by agaida**
as proposed by @jleclanche

### CODE REVIEW

**Comment by jleclanche** [comment]
Please do it in a single commit

**Comment by agaida** [comment]
no problem

**Comment by jleclanche** [comment]
Please note the "our contributors" section needs to be autogenerated from the git log and updated every release.

File: AUTHORS [hunk 1]
```diff
@@ -1,11 +1,23 @@
-Upstream authors:
-    LXQt team: http://lxqt.org
-    Razor team: http://razor-qt.org
+The LXQt panel was originally part of the Razor-qt project. It is
```
Comment (palinek) [inline AUTHORS]: "panel" ?

**Comment by agaida** [comment]
upps, fixed

**Comment by agaida** [comment]
- contributors added - list generated with  'git shortlog  -s -e' and manually removed duplicates
- i would prefer the short name in the AUTHORS file - thoughts?

**Comment by jleclanche** [comment]
Translator list is irrelevant currently as we don't use pootle in lxqt yet.

Let's also exclude the @users.noreply.github.com emails for now - just display the name, and an email iff there is a valid one and the contributors don't disagree to it.

Re short name, what do you mean?

**Comment by agaida** [comment]
short: lxqt-about
long: LXQt About Dialog
---
... and the contributors don't disagree .. - so we have to maintain the authors file by hand

**Comment by agaida** [comment]
close this for now

**Pull request closed without merging**